nucypher/config/base.py

```diff
@@ -17,40 +17,37 @@ class BaseConfiguration(ABC):
     and restoring a subclass instance from the written JSON file by passing the deserialized
     values to the subclass's constructor.
 
-    Implementation
-    ==============
+    Implementation:
 
-    `_NAME` and `def static_payload` are required for subclasses, for example:
+    `NAME` and `def static_payload` are required for subclasses, for example:
 
+    .. code::
 
-        ```
         class MyItem(BaseConfiguration):
             _NAME = 'my-item'
```

Comment:
Does this need to be `NAME` instead of `_NAME`?

Reply:
Heh - this is an odd situation - without additional docstring pre-processing - Sphinx/Autodoc thinks this `_NAME`  is invalid ( a one-sided emphasis mark ).  The *correct*  attribute name is `_NAME` in code.